.gitignore

```diff
@@ -22,6 +22,7 @@ build/
 .vagrant/
 quantum/version.h
 .idea/
+**/.directory
```

Comment:
@drashna do you know anything about this, and whether it's required?

Reply:
At least on my system (Kubuntu) is becomes necessary, otherwise i will push files like:

`[Dolphin]
SortRole=type
Timestamp=2019,4,2,22,33,38
Version=4
VisibleRoles=Details_text,Details_size,Details_modificationtime,Details_type,CustomizedDetails

[Settings]
HiddenFilesShown=true`

Reply:
Hmm, I'll defer to @jackhumbert or @skullydazed on this.

However, you can use `.git/info/exclude` to handle this in a more "localized" way.

Reply:
Should I remove this from the PR ?

Reply:
For now, yes.